docs/source/guides/getting_started.md

```diff
@@ -0,0 +1,222 @@
+if one is attached to the system hardware):
+
+```bash
+$ geth attach ~/.ethereum/goerli/geth.ipc
+> eth.accounts
+["0x287a817426dd1ae78ea23e9918e2273b6733a43d"]
+```
+
+To create a new software based Geth account:
+
+```bash
+$ geth attach ~/.ethereum/goerli/geth.ipc
+> personal.newAccount()
+...
+"0xc080708026a3a280894365efd51bb64521c45147"
+```
+
+Note that the Geth console does not return EIP-55 compliant checksum addresses, and instead will output
+the *lowercase* version of the address.  Since Nucypher requires EIP-55 checksum addresses, you will need 
+to convert your address to checksum format:
+
+```bash
+> web3.toChecksumAddress(eth.accounts[0])
+"0x287A817426DD1AE78ea23e9918e2273b6733a43D"
+```
+
+## Connecting to The NuCypher Network
+
+### Connecting Nucypher to an Ethereum Provider
+
+```python
+from nucypher.blockchain.eth.interfaces import BlockchainInterfaceFactory
+BlockchainInterfaceFactory.initialize_interface(provider_uri='~/.ethereum/goerli/geth.ipc')
+```
+
+### Ursula: Untrusted Re-Encryption Proxies
+
+When initializing an `Alice`, `Bob`, or `Ursula`, an initial "Stranger-`Ursula`" is needed to perform 
+the role of a `Teacher`, or "seednode":
+
+```python
+from nucypher.characters.lawful import Ursula
+
+seed_uri = "https://0.0.0.0:9151"
+seed_uri2 = "https://0.0.0.0:9151"
+
+ursula = Ursula.from_seed_and_stake_info(seed_uri=seed_uri, federated_only=False)
+another_ursula = Ursula.from_seed_and_stake_info(seed_uri=seed_uri2, federated_only=False)
+```
+
+Stranger `Ursula`s can be created by invoking the `from_seed_and_stake_info` method, then a `list` of `known_nodes`
+can be passed into any `Character`'s init. The `known_nodes` will inform your character of all of the nodes
+they know about network-wide, then kick-off the automated node-discovery loop:
+
+```python
+from nucypher.characters.lawful import Alice
+alice = Alice(known_nodes=[ursula, another_ursula], ...)
+```
+
+## Alice: Grant Access to a Secret
+
+```python
+from nucypher.characters.lawful import Alice, Ursula
+from nucypher.network.middleware import RestMiddleware
+from nucypher.blockchain.eth.registry import InMemoryContractRegistry
+
+ursula = Ursula.from_seed_and_stake_info(seed_uri='https://0.0.0.0:9151', federated_only=False)
+alice = Alice(known_nodes=[ursula],
+              checksum_address="0x287A817426DD1AE78ea23e9918e2273b6733a43D", 
+              registry=InMemoryContractRegistry.from_latest_publication(),
+              network_middleware=RestMiddleware())
+```
+
+Alice needs to know about Bob in order to grant access by acquiring Bob's public key's through 
+the application side channel:
+
+```python
+from umbral.keys import UmbralPublicKey
+
+verifying_key = UmbralPublicKey.from_hex(verifying_key),
+encrypting_key = UmbralPublicKey.from_hex(encryption_key)
+
+```
+
+Then, Alice can grant access to Bob:
+
+```python
+from nucypher.characters.lawful import Bob
+from datetime import datetime
```

Comment:
Btw, even `maya.timedelta(...)` would work, but it's the same timedelta as datetime's